sources/packages/backend/apps/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -94,6 +95,21 @@ export class InstitutionLocationInstitutionsController extends BaseController {
       );
     }
 
+    const isInstitutionCodeDuplicate =
+      !(await this.locationService.validateLocationCodeIsUniqueForInstitution(
```

Comment:
Following team comments from past PR, having the await assigned to a const and then doing further checks can make the code easier to read.

Reply:
Changed. Thanks!